.env

```diff
@@ -1 +1,3 @@
-REACT_APP_API_SERVER_URL=http://localhost:8001
+REACT_APP_API_SERVER_URL=databraid-transit-api-lb-1917191343.us-west-1.elb.amazonaws.com
```

Comment:
why don't these have https? 

Also there should be a .env.development file also which has the development env variables